### ISSUE TITLE
Tests fail (with HTTP::SimpleLinkChecker 1.164?)

### ISSUE THREAD

**Issue by eserte** [briandfoy/test-httpstatus#1]
Tests started to fail:
```
Can't locate object method "is_abs" via package "URI::_generic" at /home/cpansand/.cpan/build/2016102521/Mojolicious-7.09-rHWJYP/blib/lib/Mojo/UserAgent.pm line 296.
# Looks like you planned 4 tests but ran 3.
# Looks like your test exited with 255 just after 3.
t/get_status.t .... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/4 subtests 
	(less 2 skipped subtests: 1 okay)
...
Can't locate object method "is_abs" via package "URI::http" at /home/cpansand/.cpan/build/2016102521/Mojolicious-7.09-rHWJYP/blib/lib/Mojo/UserAgent.pm line 296.
# Looks like your test exited with 255 before it could output anything.
t/status.t ........ 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
```

Statistical analysis suggests that this happens with HTTP::SimpleLinkChecker 1.164:
```
****************************************************************
Regression 'mod:HTTP::SimpleLinkChecker'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    17730790868674844.00
[1='eq_1.163']        0.0000          0.0000       0.00
[2='eq_1.164']       -1.0000          0.0000    -16415522571082786.00

R^2= 1.000, N= 90, K= 3
****************************************************************
```

**Comment by briandfoy** [comment]
Where's that script for the analysis? That's pretty cool.

I'm guessing this is with the change to Mojo.

**Comment by briandfoy** [comment]
Fixed in 2d48bae96e061260201619cdf192f6a0b1cc9916 and pushing to CPAN as 2.001.

Thanks!

**Comment by eserte** [comment]
> Where's that script for the analysis? That's pretty cool.

Basically it's the same software like the one running behind analysis.cpantesters.org (sample URL: 
http://analysis.cpantesters.org/solved?distv=Test-HTTPStatus-2.001 ), only executed on my locally available test reports. So this also works if cpantesters.org is down or slow to update (like this is the case in the last weeks).
